core/templates/services/UpgradedServices.ts

```diff
@@ -212,6 +212,8 @@ import { ExplorationStatsObjectFactory } from
 import { ExplorationStatsService } from 'services/exploration-stats.service';
 import { ExplorationTaskObjectFactory } from
   'domain/improvements/ExplorationTaskObjectFactory';
+import { ExpressionEvaluatorService } from
+  'expressions/expression-evaluator.service';
```

Comment:
@srijanreddy98 Services shouldn't be added to `UpgradedServices`, right?

Reply:
This files (and others changed in my PRs) are exceptional since they are directly used by exploration editor components which haven't migrated yet. Those unmigrated services still rely on UpgradedServices, and _not_ adding them here result in >100 failures.

Reply:
Yeah, this ok for now. In a case where a lot of specs. The threshold that seems reasonable for now is >100 it blocks (or ~ 10 spec files).